<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>탐지내역</string>
  </property>
  <widget class="QListWidget" name="find_file_path">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>401</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="file_path">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>54</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>파일위치 열기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="file_info">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>130</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>파일정보 확인</string>
   </property>
  </widget>
  <widget class="QPushButton" name="file_delete">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>210</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>파일 삭제</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
